Source/PageView/PageView.swift

```diff
@@ -43,6 +43,19 @@ class PageView: UIView {
     super.init(coder: aDecoder)
     commonInit()
   }
+    
+    override func hitTest(_ point: CGPoint, with event: UIEvent?) -> UIView? {
```

Comment:
hitTest?  Please explain this function..

Reply:
It's for clicking detecting. Used by 
`guard let pageItem = pageView.hitTest(convertedLocation, with: nil) else { return }`